.circleci/config.yml

```diff
@@ -93,11 +101,26 @@ jobs:
             echo 'export ORG_GRADLE_PROJECT_mavenCentralPassword="$SONATYPE_NEXUS_TOKEN_PASSWORD"' >> "$BASH_ENV"
             echo 'export ORG_GRADLE_PROJECT_signingInMemoryKey="$SIGNING_GPG_IN_MEMORY"' >> "$BASH_ENV"
             echo 'export ORG_GRADLE_PROJECT_signingInMemoryKeyPassword="$GPG_SIGNING_KEY_PW_NEW"' >> "$BASH_ENV"
-      - run:
-          name: Publish snapshot to Maven Central
-          command: ./gradlew publishAllPublicationsToMavenCentralRepository
+      # TODO Remove this condition to allow releasing non-SNAPSHOT versions. 
```

Comment:
Hmm I'm wondering if we should just remove this and just be mindful of creating tags in the format X.Y.Z? Seems unlikely to be accidental.

Reply:
Yea we could, but this is more so I can sleep soundly at night 😛 Jokes aside, I also wanna refactor this a little bit in a follow-up PR based on a previous [comment](https://github.com/RevenueCat/purchases-kmp/pull/83#discussion_r1637869434) by César, so this is very temporary anyway.